vitest.setup.simple.ts

```
@@ -72,3 +278,76 @@ Object.defineProperty(window, "location", {
+          then: vi.fn((resolve) => resolve({ data: null, error: { message: "Invalid data" } })),
+        };
+      }
+      return createChainableMock(table, operation);
+    }),
+    in: vi.fn(() => createChainableMock(table, operation)),
+    gte: vi.fn(() => createChainableMock(table, operation)),
+    lte: vi.fn(() => createChainableMock(table, operation)),
+    order: vi.fn(() => createChainableMock(table, operation)),
+    limit: vi.fn(() => createChainableMock(table, operation)),
+    then: vi.fn((resolve) => {
+      const data = table && mockData[table] ? mockData[table] : [];
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)